### PR TITLE
Cross linking to related resources

### PR DESCRIPTION
**Pull request by udidahan**

### CODE REVIEW

**Comment by Court72** [comment]
@EdPrice-MSFT 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by EdPrice-MSFT — DISMISSED** [review]
Looks good!

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

File: docs/patterns/queue-based-load-leveling-content.md [hunk 1]
```diff
@@ -56,8 +56,10 @@ The following guidance might also be relevant when implementing this pattern:
 
 - [Choose between Azure messaging services](/azure/event-grid/compare-messaging-services). Information about choosing a messaging and queuing mechanism in Azure applications.
 
-## Related resources
+- [Asynchronous message-based communication](/dotnet/architecture/microservices/architect-microservice-container-applications/asynchronous-message-based-communication).
 
+## Related resources
+-
```
Comment (Court72) [inline docs/patterns/queue-based-load-leveling-content.md:62]:
```suggestion
```

**Review by Court72 — APPROVED** [review]

**Pull request merged**